kingpin/actors/aws/ecs.py

```diff
@@ -753,13 +756,12 @@ def _describe_service(self, service_name):
         services = response['services']
 
         # There should never be more than one service for a given name.
-        if len(services) > 1:
+        if len(services) != 1:
```

Comment:
In that case, it shouldnt be a `RecoverableActorFailure` ... it should be a custom exception and let the caller catch that exception and handle it.